assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -185,3 +185,8 @@ che.keycloak.admin_username=NULL
 # from Che database. Make sense only in case ${che.keycloak.cascade_user_removal_enabled}
 # set to 'true'
 che.keycloak.admin_password=NULL
+
+# A set of characters that are going to be used to replace `\` backslash in user name during the first login.
+# Replaced user name would be stored in Che database only. It's recommended to use DNS-compliant values.
+# Value `NULL` is turning off this feature.
+che.keycloak.username.replace_backslash_in_username_with=-
```

Comment:
might we need to replace another characters as well? I'm thinking about having it more general, and having here pairs of `[search character]:[replace with]`.

Reply:
done